README.md

```diff
@@ -1,3 +1,18 @@
+## Personal Entry
+
```

Comment:
Really nice reflection! :)

Reply:
It's really cool that you're thinking about _when_ to refactor and reflecting on your decision there.  This will be really useful in the long term.